packages/serialize-error/test/lib/index.spec.js

```diff
@@ -84,6 +85,30 @@ describe('@dotcom-reliability-kit/serialize-error', () => {
 			});
 		});
 
+		describe('when the error has a `cause` property', () => {
+			it('includes the root cause error instance in serialized form in the serialized error properties', () => {
+				const rootCauseErrorInstance = new Error(
+					'mock root cause error message'
+				);
+				error.cause = rootCauseErrorInstance;
+				expect(serializeError(error)).toMatchObject({
+					cause: serializeError(rootCauseErrorInstance)
+				});
+				expect(serializeError(error).cause).toMatchObject({
+					message: 'mock root cause error message'
+				});
```

Comment:
I could have gone to town on testing the serialisation of the `cause` error to provide the same coverage that is given for the subject (i.e. containing) error, e.g. when the `cause` value is the result of `serialiseError()` being called with:
- an error object
- en error-like object
- a string
- a number
- an array

However, the `serialiseError()` is already being tested so thoroughly by the subject error, so ensuring that the root cause error `message` value is as expected felt to be a sufficient measure for testing this.